Libraries/Utilities/UserServiceExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+using OrchardCore.Users.Models;

```

Comment:
Ezek nem ABC-sorrendben vannak, hanem előbb van a framework, pedig az .editorconfigban az előbbi van. Szal tanítsd ezt meg a Ridernek is légyszi.

Reply:
Eh pedig ezt onnan szedi (mindig erősebb, mint a ReSharper), de akkor lehet valami elállítódott.